contrib/kafka/samples/README.md

```diff
@@ -86,6 +128,30 @@ and an Event Display Service.
 
 1. Modify `contrib/kafka/samples/event-source.yaml` accordingly with bootstrap
    servers, topics, etc...
+
+   NOTE: If using an internal kafka cluster, you may need to ensure
+you've specified the correct variables set in any Kafka Topic and Event Source:
+
+    ``` shell
+    $ cat contrib/kafka/samples/kafka-event-source.yaml
```

Comment:
perhaps we name this `kafka-sample-source` ?

Reply:
Sure.  As we discussed on slack.  It's probably best to not introduce another file with such overlap with the generic event-source.  Instead, opting to just show an example in the README.md that will demonstrate our point.